dockstore-webservice/pom.xml

```diff
@@ -53,22 +53,22 @@
         <dependency>
             <groupId>io.dockstore</groupId>
             <artifactId>swagger-java-quay-client</artifactId>
-            <version>2.0.1</version>
```

Comment:
Many of our own libraries are bumped with new versions on artifactory with the new jakarta libraries.